MonitoringSystem/Client/Types/BaseType.py

```diff
@@ -19,6 +21,9 @@ class BaseType( object ):
   :param int dataToKeep: Data retention. We keep all data by default.
   :param dict mapping: We can specify the mapping of the documents. It is used during the creation of an index.
                        Note: If you do not want to be analysed a string, you have to set the mapping
+  :param str period: We can define, which kind of index we want to create. By default we use daily indexes. But we can use monthly indexes.
+                    Supported values: day, month
+  
```

Comment:
This refers to the elastic search index. So we can create daily and monthly indexes.